allennlp/data/tokenizers/sentence_splitter.py

```diff
@@ -0,0 +1,81 @@
+from typing import List
+from overrides import overrides
+from allennlp.common import Registrable
+from allennlp.common.util import get_spacy_model
+from allennlp.data.tokenizers.token import Token
+
+
+class SentenceSplitter(Registrable):
+    """
+    A ``SentenceSplitter`` splits strings into sentences. There are two variants of sentence splitting,
+    one that uses rules (ie punctuation), and another that uses a dependency parse of the text to infer
+    sentence boundaries.
+    """
+    default_implementation = 'spacy_rulebased'
+
+    def batch_split_sentences(self, texts: List[str]) -> List[List[Token]]:
+        """
+        This method lets you take advantage of spacy's batch processing.  
+        Default implementation is to just iterate of the sentences and call ``split_sentences``.
+        """
+        return [self.split_sentences(text) for text in texts]
+
+    def split_sentences(self, text: str) -> List[Token]:
```

Comment:
Is this really a `Token` object?  Should it be?  I'm pretty sure it's not - it's a sentence, not a token.  I'd expect this to be `List[str]`.  Is there some additional information from spacy that you're hoping to keep around that makes you use `Token` here?

Reply:
Yeah, this is a typo, should be List[str].